typescript-stl/typescript-stl.d.ts

```diff
@@ -3629,8 +3622,8 @@ declare namespace std {
      * <p> All objects thrown by components of the standard library are derived from this class.
      * Therefore, all standard exceptions can be caught by catching this type by reference. </p>
      *
-     * <p> <a href="http://samchon.github.io/typescript-stl/api/assets/images/design/exceptions.png" target="_blank">
-     * <img src="http://samchon.github.io/typescript-stl/api/assets/images/design/exceptions.png" style="max-width: 100%" /> </a> </p>
+     * <p> <a href="D:/Homepage/samchon.github.io/typescript-stl/images/class_diagram/exceptions.png" target="_blank">
+     * <img src="D:/Homepage/samchon.github.io/typescript-stl/images/class_diagram/exceptions.png" style="max-width: 100%" /> </a> </p>
```

Comment:
and in other places in this file

---
In reply to: [74475584](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/10557#discussion_r74475584) [](ancestors = 74475584)